analyzer/src/funTest/assets/projects/external/spdx-tools-python-expected-output.yml

```diff
@@ -27,8 +27,47 @@ project:
     dependencies:
     - id: "PyPI::ply:3.11"
     - id: "PyPI::rdflib:5.0.0"
-    - id: "PyPI::six:1.16.0"
+      dependencies:
```

Comment:
One should probably note that this restores the expected test result to (almost) how it was before 16c4d66b0c6f534ced9e9220761895dce114a90b. So that's good.